src/main/java/com/plivo/api/models/message/MessageMeta.java

```diff
@@ -0,0 +1,19 @@
+package com.plivo.api.models.message;
+
+import java.lang.NoSuchMethodException;
+import com.plivo.api.models.base.Meta;
+
+public class MessageMeta extends Meta {
+
+    public MessageMeta(Meta m) {
+        limit = m.getLimit();
```

Comment:
Not sure, do we not needed to initialise the MessageMeta class variables here? like int limit, etc etc

Reply:
Not required, because this class is a subclass of Meta class which already has these defined